DataSetExplorer/DataSetBuilder/DataSetAnalysisService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using DataSetExplorer.DataSetBuilder.Model;
+using DataSetExplorer.DataSetSerializer;
+
+namespace DataSetExplorer
+{
+    class DataSetAnalysisService : IDataSetAnalyzer
+    {
+        public void FindInstancesWithAllDisagreeingAnnotationsUseCase(string dataSetPath, string outputPath)
+        {
+            var dataset = LoadDataSet(dataSetPath);
+            var exporter = new TextFileExporter(outputPath);
+            exporter.ExportInstancesWithAnnotatorId(dataset.GetInstancesWithAllDisagreeingAnnotations());
+        }
+
+        public void FindInstancesRequiringAdditionalAnnotationUseCase(string dataSetPath, string outputPath)
```

Comment:
Return Result<string> and if you will append UseCase do so consistently (e.g., in DataSetAnalyzer/Exporter).